src/com/connectsdk/sampler/MainActivity.java

```diff
@@ -46,18 +46,23 @@
 
 public class MainActivity extends ActionBarActivity implements ActionBar.TabListener {
 
-    private ConnectableDevice mTV;
-    private AlertDialog dialog;
-    private AlertDialog pairingAlertDialog;
-    private AlertDialog pairingCodeDialog;
-    private DevicePicker dp; 
+    ConnectableDevice mTV;
+    AlertDialog dialog;
+    AlertDialog pairingAlertDialog;
+    AlertDialog pairingCodeDialog;
+    DevicePicker dp; 
 
-    private MenuItem connectItem;
+    MenuItem connectItem;
 
     SectionsPagerAdapter mSectionsPagerAdapter;
 
     ViewPager mViewPager;
-
+    ActionBar actionBar;
+    
+    ActionBar getSupportedActionBar(){
+		return getSupportActionBar();
```

Comment:
There is no particular reason to add a this as method, So it can be replaced with
ActionBar actionBar = getSupportActionBar();